requirements.txt

```diff
@@ -15,7 +15,7 @@ imageio[ffmpeg]
 trimesh
 git+https://github.com/NVlabs/nvdiffrast.git
 # deepfloyd
-git+https://github.com/facebookresearch/xformers.git@main#egg=xformers
+xformers
```

Comment:
Why this change?

Reply:
IIRC `git+https://github.com/facebookresearch/xformers.git@main#egg=xformers` wouldn't install on our cluster due to an incompatibility between the particular pytorch and CUDA toolkit versions, while `xformers` installed fine.

Reply:
However as indicated in the [xformers README](https://github.com/facebookresearch/xformers#installing-xformers), installing directly from `pip` requires `torch>=2.0.0`. Which PyTorch version are you using?

Reply:
Personally I'm using torch 2.x but IIRC, I previously tried installing pytorch 1.x with your instructions, and that's when I bumped into that xformers incompatibility. @voletiv can you check on your side if you can reproduce the problem with 1.x? Otherwise we can revert that particular change.

Reply:
I couldn't install tinycudann itself in torch 1.12, so the code doesn't run with 1.x in my case.

Reply:
<img width="447" alt="Screenshot 2023-05-29 at 16 26 05" src="https://github.com/threestudio-project/threestudio/assets/22424247/e66f0892-e9f6-400f-a390-a0eac823898e">